protobuf/signalling.proto

```diff
@@ -141,6 +141,25 @@ enum TransportType {
     Null = 0;
     Min = 1;   // Send a 32-byte HMAC id to let the station distinguish registrations to same host
     Obfs4 = 2; // Not implemented yet?
+    Webrtc = 3; // DTLS transport, yay
+}
+
+message DeflatedSDP {
+    optional uint32 sdp_type = 1; // TODO: Single byte (uint8)
+    // the IP is represented as a 16-byte datatype.
+    optional uint64 ip_upper = 2; // ip := upper << 64 | lower
+    optional uint64 ip_lower = 3;
+    optional uint32 composed_info = 4;
+}
+
+message WebrtcRandseed {
+    optional string seed = 1;
+    optional string shared_secret = 2;
```

Comment:
We should probably not send the shared secret in the clear like that, no? Is this derived from the seed? If so, then no need to send it in this protobuf